core/frontend/src/index.css

```
@@ -1,4 +1,128 @@
+body.interest-intuition {
+  --primary-accent: #10a37f;
+  --glow-intensity: rgba(16, 163, 127, 0.3);
+}
+
+/* --- Core Styles --- */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', -apple-system, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)